bin/tictactoe

```diff
@@ -1,3 +1,48 @@
 #!/usr/bin/env ruby
-
 require_relative '../config/environment'
+require 'pry'
+system 'clear'
+puts "Welcome to Tic Tac Toe!"
+puts ""
+puts " 1 | 2 | 3    "
+puts "----------- "
+puts " 4 | 5 | 6    "
+puts "----------- "
+puts " 7 | 8 | 9    "
+puts ""
+puts "One moment please."
+loop do 
+  sleep 5
+  system 'clear'
+  begin 
+    puts "How many human players?(0-2)"
+    answer = gets.chomp.to_i
+  end until [0,1,2].include?(answer)
+  system 'clear'
+  if answer == 0
+    game = Game.new(Computer.new('X'), Computer.new('O'))
+  elsif answer == 1
+    begin
+      puts "Do you want to go first or second[1,2]?"
+      one_or_two = gets.chomp
+    end until ["1","2"].include?(one_or_two)
+    begin
+      x_or_o = ['X','O']
+      puts "Do you want to be X or O?"
+      answer = gets.chomp.upcase
+      system 'clear'
+    end until x_or_o.include?(answer)
+      x_or_o.delete(answer)
+      game = Game.new(Human.new(answer), Computer.new(x_or_o[0])) if one_or_two == "1"
+      game = Game.new(Computer.new(x_or_o[0]), Human.new(answer)) if one_or_two == "2"
+  else
+    game = Game.new
+  end
+  game.play
+  begin
+    puts "Play again?(y/n)"
+    answer = gets.chomp.downcase
+  end until %w{y n}.include?(answer)
+  break if answer == 'n'
+end
+puts "Thanks for playing!  Goodbye!"
```

Comment:
I might have put this logic within an object, like `TicTacToeCLI` so that the entirety of this file could be:

`TicTacToeCLI.new`

`TicTacToeCLI#initialize` would handle the player logic and options and then start the game.